homeassistant/components/weather/openweathermap.py

```diff
@@ -47,6 +49,7 @@
     vol.Optional(CONF_LATITUDE): cv.latitude,
     vol.Optional(CONF_LONGITUDE): cv.longitude,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_FORECAST): cv.boolean
```

Comment:
Please specify defaults in config validation.

Any reason why we would not just turn this always on ?